### ISSUE TITLE
Adding Support for DVR163 / EseeCloud

### ISSUE THREAD

**Issue by wes1993** [AlexxIT/go2rtc#511]
Hello,
thanks a lot for your wonderful work!!! 
I'm trying to config my cameras with this software but unfortunately i can't find a way :-(
This is the command i use to access the stream of my cameras using FFMPEG:
ffplay -headers "Authorization: Basic YWRtaW46" -i "http://192.168.0.50/livestream/11"
Here is the screen:
![image](https://github.com/AlexxIT/go2rtc/assets/52070589/1bb9d2d6-a296-4241-9a64-9e8aea363bb4)
The output of FFPROBE:
![image](https://github.com/AlexxIT/go2rtc/assets/52070589/af3ce4a4-1f98-472c-b304-a449064db9d6)

So my question is how can i access the stream of my cameras? with FFPROBE we know that the codecs is 
Video: h264 (High), yuv420p(tv, bt470bg, progressive), 2304x1296, 25 fps, 30 tbr, 1200k tbn

Just an addition info:
Basically we can access this chinese camera using two ways, the first with the custom header (less stable) explained above the other using a custom handshake that we can simple reply using the python script here (more stable but difficult to implement):
https://github.com/pgross41/hassio-addons/blob/4c5f7f1afdf4c02481469bf1e81c664f517c8614/dvr163/app/stream.py

At the end, if we can add support for this cameras using the python script included in go2rtc this is really wonderful but i know is not simple so the other way also could be a solution but i can't find a way to implement the custom header.

I hope in your help
Best Regards
Stefano

**Comment by AlexxIT** [comment]
Can you send me traffic dump? Better all you can get with `stream.py` without any additional processing.
I want to see the transport format of video and audio.

**Comment by wes1993** [comment]
@AlexxIT thanks a lot for your reply,
sorry for my delay very busy at works this days... sure tomorrow i will send you the traffic dump.

Thanks a lot
Best Regards
Stefano

**Comment by wes1993** [comment]
Dear @AlexxIT, i have attached the dump of my cameras and the screen of ffplay:
![image](https://github.com/AlexxIT/go2rtc/assets/52070589/785010c5-50ea-46ed-8af2-fb86645e0c8a)

If we can implement this probably we cover the 90% of the chinese cameras on the market, there is also a thread here:
https://community.home-assistant.io/t/support-for-security-camera-system-via-nvr-eseecloud-dvr163/35502/77

[Dump.zip](https://github.com/AlexxIT/go2rtc/files/12030368/Dump.zip)

I hope in your miracle
Best Regards
Stefano

P.S. I have got two dump, one using the save as button, the other (DumpPacket) using the export packet functionality.

**Comment by wes1993** [comment]
Hi @AlexxIT,
I have written to you in Telegram, thanks a lot for you reply/help :)

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/go2rtc/releases/tag/v1.6.1
